aws/services/ses/id.ftl

```diff
@@ -36,4 +36,28 @@
     [#return formatAccountResourceId(
                 AWS_SES_RECEIPT_FILTER_RESOURCE_TYPE,
                 extensions)]
-[/#function]
\ No newline at end of file
+[/#function]
+
+[#assign AWS_SES_CONFIGSET_RESOURCE_TYPE = "sesconfigset" ]
+[@addServiceResource
+    provider=AWS_PROVIDER
+    service=AWS_SIMPLE_EMAIL_SERVICE
+    resource=AWS_SES_CONFIGSET_RESOURCE_TYPE
+/]
+
+[#assign AWS_SES_CONFIGSET_DEST_RESOURCE_TYPE = "sesconfigdest" ]
+[@addServiceResource
+    provider=AWS_PROVIDER
+    service=AWS_SIMPLE_EMAIL_SERVICE
+    resource=AWS_SES_CONFIGSET_DEST_RESOURCE_TYPE
+/]
+
+[#function formatSESConfigSetId ]
+    [#return formatAccountResourceId(AWS_SES_CONFIGSET_RESOURCE_TYPE) ]
+[/#function]
+
+[#function formatSESConfigDestId extensions...]
```

Comment:
Suggest using `Destination` - we tend to err on the more verbose side of naming

Reply:
Cleaned up along with below